### PR TITLE
Canary Release docs diagram update

### PR DESCRIPTION
**Pull request by jehawley**
Update diagrams used in Canary Release docs. It looked to me like previous diagram updates didn't remove old images, but if that's safe then I'll do that instead.
BOT NOTES: 
resolves https://github.com/solo-io/gloo/issues/3293

### CODE REVIEW

File: docs/content/guides/traffic_management/destination_types/canary/_index.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ As application architecture evolves from monolithic to microservices, the strate
 What is a Canary Release?
 Canary release is an approach for safer application delivery, specifically the delivery of new software to end users without disrupting or interrupting their experience. The term ["canary release"](https://blog.christianposta.com/deploy/blue-green-deployments-a-b-testing-and-canary-releases/) is to test whether the intended changes behave well and without issues when taking a small fraction (think, 1% ) of the traffic. If this canary deployment starts to misbehave (as judged by external metric collection like request throughput or latency) then we immediately roll back by directing traffic back to the last known-working deployment. Techniques like Canary fall within the practice of Progressive Delivery which allow organizations to better manage their risk by slowing down and controlling how many end users have access to the new software as it is deployed to production.
 
-![Canary traffic]({{% versioned_link_path fromRoot="/img/solo-canary.png" %}})
+![Canary traffic]({{% versioned_link_path fromRoot="/img/solo-canary-1.png" %}})
```
Comment (kdorosh) [inline docs/content/guides/traffic_management/destination_types/canary/_index.md:12]: why add the `-1` suffix? to force an image reload for users who have the old image in cache?

Comment (jehawley) [reply docs/content/guides/traffic_management/destination_types/canary/_index.md:12]: I couldn't find any examples anywhere of us ever deleting or updating an image in docs without changing the name (and I couldn't find any instance of image deletion period; I bet there are some old assets we can clean up). The last time I used check-in images for a project we also had a strict "never delete, never modify, only add" policy (whose purpose I did not really understand). If we think it's safe to just replace, that would be cleaner.

**Comment by jehawley** [comment]
> edit: nevermind it's an image with a slightly different name

I saw that, and assumed it was a different image. But it looks identical to me. Should I ask... Betty(?) if we should replace that one too?

File: docs/content/guides/traffic_management/request_processing/shadowing/_index.md [hunk 2]
```diff
@@ -10,7 +10,7 @@ When releasing changes to a service, you want to finely control how those change
 ## What is traffic shadowing with Gloo?
 Traffic shadowing with Gloo makes a copy of an incoming request and proxies the real request to the appropriate backend (the normal request path) and sends the copy to another upstream. The copied message is ignored for failures or responses. In this case, you can deploy `v2` of a service and shadow traffic to it *in production* without affecting user traffic. This ability to shadow is incredibly important because it allows you to begin your release or canary process with zero production impact. 
 
-![Gloo traffic shadowing diagram]({{% versioned_link_path fromRoot="/img/gloo-traffic-shadowing.png" %}})
+![Gloo traffic shadowing diagram]({{% versioned_link_path fromRoot="/img/traffic-shadowing.png" %}})
```
Comment (kdorosh) [inline docs/content/guides/traffic_management/request_processing/shadowing/_index.md:13]: didn't we remove `/img/traffic-shadowing.pn` in this PR?

Comment (kdorosh) [reply docs/content/guides/traffic_management/request_processing/shadowing/_index.md:13]: also i believe `img/solo-canary.png` is referenced elsewhere so we can't remove it

Comment (jehawley) [reply docs/content/guides/traffic_management/request_processing/shadowing/_index.md:13]: We're not deleting them - we're replacing them with different images. I don't know why the diff here shows them both as deleted, but they appear in my local checkout?

**Pull request merged**